gnxi_tester/web/upload.go

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2020 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package web
+
+import "net/http"
+
+func handleUpload(w http.ResponseWriter, r *http.Request) {
+
+}
+
+func handleDelete(w http.ResponseWriter, r *http.Request) {
```

Comment:
Should this be in a separate module, if not I would change the name also. It might not be obvious to readers that upload contains both.